CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+## Unreleased
+
+## Version 0.0.1 (2022-02-01)
```

Comment:
Are you suggesting we tag this commit as 0.0.1? If not, you'll need to lose this heading (and leave only the `## Unreleased`) line in, and also set `current_version` to 0.0.0 in `.bumpversion.cfg`.

Reply:
OK, I'll fix :hammer_and_wrench: